soc/sifive/sifive_freedom/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) 2024 Nordic Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
v1 files should be removed in the same commit

Reply:
Should be removed in the same commit now